src/main/scala/shade/memcached/internals/SpyMemcachedIntegration.scala

```diff
@@ -379,6 +379,38 @@ class SpyMemcachedIntegration(cf: ConnectionFactory, addrs: Seq[InetSocketAddres
     prepareFuture(key, op, promise, timeout)
   }
 
+  def realAsyncMutate(key: String, by: Long, mutator: Mutator, default: Option[Long], exp: Duration, timeout: FiniteDuration)(implicit ec: ExecutionContext): Future[Result[Long]] = {
+    val promise = Promise[Result[Long]]()
+    val result = new MutablePartialResult[Long]
+
+    val expiry = default match {
+      case Some(d) => expiryToSeconds(exp).toInt
+      case None => -1 // expiry of all 1-bits disables setting default in case of nonexistent key
+    }
+    val op: Operation = opFact.mutate(mutator, key, by, default.getOrElse(0L), expiry, new OperationCallback {
+      def receivedStatus(opStatus: OperationStatus) {
+        if (statusTranslation.isDefinedAt(opStatus))
```

Comment:
No problem at all; I can appreciate wanting to be consistent with the rest of the file :)